e2etests/tests/advertisement.go

```diff
@@ -107,6 +109,31 @@ var _ = ginkgo.Describe("Advertisement", func() {
 
 			ginkgo.By("validating")
 			p.validate(peersV4, peersV6, nodes)
+
+			if p.splitCfg == nil {
+				return
+			}
+
+			ginkgo.By("Cleaning before retesting with the config splitted")
```

Comment:
I understand this is more compact, but we are packing two tests in one for the sake of writing less entries. Can we find a better way? It will be even easier to understand if a test failed in it's regular or split variant